src/repositories/user.ts

```diff
@@ -0,0 +1,52 @@
+import { User } from '../models/User';
+import { CreateUser } from '../models/CreateUser';
+import { v4 as uuidv4 } from 'uuid';
+import { UpdateUser } from '../models/UpdateUser';
+import { ApplicationError } from '../models/application-error';
+
+const store = new Map<string, User>();
+
+export function getByIdFromDB(id: string): User {
+  const user: User | undefined = store.get(id);
+  if (user) {
+    return user;
+  }
+
+  throw new Error(`user ${id} not found`);
```

Comment:
function should be as predictable as possible, for this reason it should return expected value but in current implementation `undefined` is a value that could be returned from function when error is raised. 

I propose to return "-1" if user was not found and remove the `throw`.

As a rule, error throwing can be used on controller level only. In all the rest cases the return statement should define the return state.